instrumentation/jdbc/bootstrap/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("otel.javaagent-bootstrap")
+}
+
```

Comment:
I think this file would be a good place to document why DbInfo needs to be in the bootstrap class loader, maybe walking through an example of how the wrong DbInfo class can get picked up without this